h/static/scripts/vendor/frame-rpc.js

```diff
@@ -0,0 +1,85 @@
+var isarray = require('isarray');
```

Comment:
The upstream license needs to be reproduced at the top of this file - see https://github.com/substack/frame-rpc/blob/master/LICENSE